lib/manager/travis/package.js

```diff
@@ -2,20 +2,60 @@ const is = require('@sindresorhus/is');
 const { isEqual } = require('lodash');
 const { getPkgReleases } = require('../../datasource');
 const { isVersion, maxSatisfyingVersion } = require('../../versioning/semver');
+const nodeJsSchedule = require('./data/node-js-schedule.json');
 
 module.exports = {
   getPackageUpdates,
 };
 
-// Start version numbers as integers for correct sorting
-const policies = {
-  all: [8, 10, 11, 12],
-  lts: [8, 10],
-  active: [10, 12],
-  lts_active: [10],
-  lts_latest: [10],
-  current: [12],
-};
+let policies;
+
+function generatePolicies() {
+  policies = {
+    all: [],
+    lts: [],
+    active: [],
+    lts_active: [],
+    lts_latest: [],
+    current: [],
+  };
+
+  const now = new Date();
+
+  for (const [vRelease, data] of Object.entries(nodeJsSchedule)) {
```

Comment:
Shouldn't we sort the versions, because json property order is not guaranteed?

Reply:
I thought that the v8 ending now does, e.g. http://2ality.com/2015/10/property-traversal-order-es6.html